travis/install-common-deps.sh

```diff
@@ -2,17 +2,15 @@
 
 set -ex
 sudo mkdir -p /travis-install
-sudo chown travis /travis-install
+if [[ -z $GITHUB_WORKFLOW ]]; then
```

Comment:
Added some conditional logic to only run stuff when `GITHUB_WORKFLOW` is not set.